playwright-tests/tests/settings/theme.spec.js

```diff
@@ -0,0 +1,440 @@
+import path from "path";
+import { expect } from "@playwright/test";
+import { NearSandbox, parseNEAR } from "../../util/sandbox.js";
+import { test } from "../../util/test.js";
+import { setupTestEnvironment } from "../stake-delegation/test-helpers.js";
```

Comment:
we should move this function to `util`
I had it inside stake-delegation as it was just used in that tests, but now it's getting used in settings as well, so makes sense to have inside `util` folder